oci_store.go

```diff
@@ -126,13 +134,21 @@ func (s *OCIStore) FetchBlobContent(ctx context.Context, _ string, desc ocispec.
 // FetchImageManifest returns the referenced image manifest as given by the
 // descriptor.
 func (s *OCIStore) FetchImageManifest(ctx context.Context, _ string, desc ocispec.Descriptor) (*ocispec.Manifest, error) {
-	manfiestBytes, err := content.FetchAll(ctx, s.store, desc)
+	manifestBytes, err := content.FetchAll(ctx, s.store, desc)
 	if err != nil {
 		return nil, fmt.Errorf("failed to fetch manifest: %w", err)
 	}
+	return parseImageManifest(manifestBytes)
+}
+
+// parseImageManifest parses the given manifest bytes into an image manifest.
+func parseImageManifest(manifestBytes []byte) (*ocispec.Manifest, error) {
```

Comment:
@binbin-li This function is invoked by all store implementations. Should this be moved to the `Executor` logic while `FetchImageManifest` should return `([]byte, error)`?

cc: @Two-Hearts

Reply:
Resolved by #33